.github/workflows/licensed.yml

```diff
@@ -0,0 +1,20 @@
+name: Licensed
+
+on:
+  push: {branches: main}
+  pull_request: {branches: main}
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    name: Check licenses
+    steps:
+      - uses: actions/checkout@v2
+      - run: npm ci
+      - name: Install licensed
+        run: |
+          cd $RUNNER_TEMP
+          curl -Lfs -o licensed.tar.gz https://github.com/github/licensed/releases/download/2.12.2/licensed-2.12.2-linux-x64.tar.gz
+          sudo tar -xzf licensed.tar.gz
+          sudo mv licensed /usr/local/bin/licensed
```

Comment:
Is there no `latest` link that we can use?  Or are we pinned to this version intentionally?

Reply:
https://docs.github.com/en/github/administering-a-repository/linking-to-releases

Reply:
the problem is the asset name has the version in the zip. We shouldn't really need to upgrade this that often, and if we do, its probably due to the workflow failing so we will get visibility into needing an upgrade.